### PR TITLE
Code Sleep Python first timers only issue added

### PR DESCRIPTION
**Pull request by prateekiiest**
Added a new project Code Sleep Python

https://github.com/prateekiiest/Code-Sleep-Python

### CODE REVIEW

**Comment by prateekiiest** [comment]
@MunGell reviews please :)

**Comment by prateekiiest** [comment]
Any updates?

File: README.md [hunk 1]
```diff
@@ -141,6 +141,8 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Concept to Clinic](https://github.com/concept-to-clinic/concept-to-clinic/labels/good%20first%20issue) _(label: good first issue)_ <br> Aims to develop an end-to-end application for lung cancer detection that connects the predictive power of machine learning with functional software tested against errors and a clean user interface focused on clinical use.
 - [Zulip](https://github.com/zulip/zulip/labels/good%20first%20issue) _(label: good first issue)_ <br> Powerful open source group chat.
 - [cookiecutter](https://github.com/audreyr/cookiecutter/labels/low-hanging-fruit) _(label: low-hanging-fruit)_ <br> A command-line utility that creates projects from cookiecutters (project templates). E.g. Python package projects, jQuery plugin projects.
+- [Code Sleep Python](https://github.com/prateekiiest/Code-Sleep-Python/labels/first-timers-only) _(label: first-timers-only)_ <br> A curated list of sub-projects related to desktop applications, games and machine learning applications.
+
```
Comment (spnraju) [inline README.md]: I think you need to remove the extra line on line number 145

**Comment by prateekiiest** [comment]
Done @spnraju

**Comment by prateekiiest** [comment]
Any updates @spnraju

**Comment by MunGell** [comment]
Hi @prateekiiest 

Looks like you don't have any open issues with this label at the moment.
From the history you only had 2 of them. I will close this PR, but if you will have a stronger need/offer of easy issues, please feel free to create a new PR.

Thanks!

**Pull request closed without merging**